src/components/frame/Tag.vue

```diff
@@ -85,7 +85,7 @@ export default class Tag extends Mixins(GlobalMixin) {
   getPreviewUrl(photo: IPhoto) {
     if (this.isFace) {
       return generateUrl(
-        "/apps/memories/api/faces/preview/" + this.data.fileid
+        "/apps/memories/api/recognize/people/preview/" + this.data.fileid
```

Comment:
You do need to call the `facerecognition` API here. I assume that's something WIP?

Reply:
> You do need to call the facerecognition API here. I assume that's something WIP?

You say Faceregnition own api? Just changed it, to be consistent and then use an "/apps/memories/api/facerecognition/people/preview/" endpoint..

If we are going to reuse and not duplicate components as you mentioned, here i will add a switch case to select the correct endpoint.

P.S: The api entry "/apps/memories/api/facerecognition/people/preview/" already works, and my tests are changing the method in the routes.php file, and other small changes. I'll comment on it later. 🤔